README.md

```diff
@@ -88,4 +88,12 @@ To run the end to end tests you will need
 4. To run the e2e tests >> poetry run pytest todo_app/tests_e2e
 
 
+To build the docker images run the below 
+build --target development --tag todo-app:dev .
+build --target production --tag todo-app:prod .
 
+To run the docker images 
+docker run --env-file <env_file> -dp 5000:5000 todo-app:dev
```

Comment:
Part 2 of the exercise asks you to make sure hot reloading works when you run the development container. You'll need to mount the current directory as part of this command, a bit like you do in the docker-compose file